plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -3148,8 +3123,8 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
             path="/subscriptions/{subscriptionId}/providers/Microsoft.Network/ExpressRoutePortsLocations",
             path_parameters=["subscriptionId"],
             query_parameters=["api-version"],
-            compared_property=lambda r: r["name"],
-            binding_property=lambda r: r["id"],
+            first_property=lambda r: r["name"],
+            second_property=lambda r: r["id"],
```

Comment:
This call is not required - it fetches all express route locations, which have been already fetched and are available in the graph.